### DISCUSSION TITLE
ERROR: Stack walk

### DISCUSSION THREAD

**Discussion by colorado-bob** [dashingsoft/pyarmor#1325] [Q&A]
Anybody seen this error before or what causes it?

Java frame anchors for the failing thread 0x00007fa0ac000b80
Fatal error: Stack walk must walk only frames of known code

The pyarmor obfuscated code generates this error.  
From what the web found, stack walk is trying to do a stack trace from an error?

the error output is not helpful since its all hex so i cannot tell what causes the issue.
the print "# pyarmor: " for debugging probably wouldnt help since i would not know where to start.
the code uses third party libraries like pandas and saxon but the error doesnt point to these libraries.

is there a way to find/debug what is causing this?

**Comment by MortenKaae** [comment]
@colorado-bob did you ever find a solution to this? I'm facing the same issue using Saxon with Django.
